e2e/cli-plugins/testdata/docker-help-helloworld.golden

```diff
@@ -1,8 +1,11 @@
 
-Usage:	docker helloworld COMMAND
+Usage:	docker helloworld [OPTIONS] COMMAND
```

Comment:
nit: looks like `COMMAND` is optional for this plugin (the top-level `helloworld` is a functional command). 

```suggestion
Usage:	docker helloworld [OPTIONS] [COMMAND]
```


We could add a demo plugin that does not have a top-level command (so that it prints the `help` output if no subcommand is given)

(not a show-stopper, just something I noticed)

Reply:
> nit: looks like COMMAND is optional for this plugin (the top-level helloworld is a functional command).

Yeah, it looks like the `COMMAND` here comes from the existing `usageTemplate` so this would be presented the same way for a builtin command with similar behaviour.